src/Core/Mews.Fiscalizations.Core/Model/Countries/Countries.cs

```diff
@@ -116,7 +116,7 @@ static Countries()
                 Indonesia = NonEuropean("ID"),
                 Iran = NonEuropean("IR"),
                 Iraq = NonEuropean("IQ"),
-                Ireland = European("IE", "^(IE)?[0-9][0-9A-Z\\+\\*][0-9]{5}L$"),
+                Ireland = European("IE", "^(IE)?[0-9A-Z]{8}(?:[0-9A-Z]{1})?$"),
```

Comment:
OK, I accept the change of requirements, but why do we have the non-capturing `?:` in the middle?

Reply:
Yea i missed that, its redundant.